views/appView.js

```diff
@@ -0,0 +1,31 @@
+var AppView = Backbone.View.extend({
+  el: $('body'),
+
+  events: {
+    'click .submit': 'searchVideo'
+
+  },
+  initialize: function (){
+    console.log('view initialized')
+    // this.listenTo(this.model.get('videos'), 'reset', this.renderPage)
+  },
+
+  renderPage: function () {
+
```

Comment:
Where is this function?  Why didn't you attempt this part of it?  I think your unnecessary instantiations at the end of the files threw you off the right track here.